### ISSUE TITLE
create swagger

### ISSUE THREAD

**Issue by julienBelinga** [paye-ton-kawa/API_webshop#2]

**Comment by paulawritesit** [comment]
Hello, I'm interested in contributing to open-source documentation. Can I help with this issue?

**Comment by julienBelinga** [comment]
Yes you can if you want i going to add you to the project. You will need to create your branch to start
